var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -16,7 +16,7 @@ class PyBglibpy(PythonPackage):
 
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('neuron+python~mpi', type='run')
+    # depends_on('neuron+python~mpi', type='run')
```

Comment:
As discussed yesterday, lets keep this back so that people's script don't break?